pkg/reconcile/pipeline/infinispan/handler/manage/upgrades.go

```diff
@@ -204,10 +204,15 @@ func GracefulShutdownUpgrade(i *ispnv1.Infinispan, ctx pipeline.Context) {
 
 		ctx.Requeue(
 			ctx.UpdateInfinispan(func() {
+				// Ensure that CR has all the operator required labels
+				if !subMapOf(ctx.DefaultLabels(), i.ObjectMeta.Labels) || !subMapOf(ctx.DefaultAnnotations(), i.ObjectMeta.Annotations) {
```

Comment:
Do we need the `subMapOf` checks, can't we just always call `i.ApplyOperatorMeta(ctx.DefaultLabels(), ctx.DefaultAnnotations())`?